src/applications/gi/components/profile/AdditionalInformation.jsx

```diff
@@ -158,6 +158,18 @@ export class AdditionalInformation extends React.Component {
           &nbsp;
           {institution.stemIndicator ? 'Yes' : 'No'}
         </div>
+        <div>
+          {institution.vrrap === null ? (
```

Comment:
Why not just check for !== null and no else needed?

Reply:
For AC 1c c. "Veteran Rapid Retraining Assistance Program (VRRAP):" is not displayed if the institution facility code is not included in the VRRAP provider data. Then a technical note `AC1c will be NULL from the backend data source.`